### DISCUSSION TITLE
3 values ​​for some entries

### DISCUSSION THREAD

**Discussion by openKimchi** [revk/xESP32-Faikout#164] [General]
In the Json object that I get via MQTT, I have 3 values ​​for some entries (env, home, fanrpm, comp, outside, liquid).
Why is this so and what is the current value?

the json looks like this:
{
  "protocol": "S21",
  "ts": "2023-12-11T15:50:00Z",
  "env": [
    21.98,
    22.02,
    22.05
  ],
  "online": true,
  "home": [
    25,
    25.12,
    25.5
  ],
  "heat": true,
  "fanrpm": [
    0,
    85,
    730
  ],
  "comp": [
    0,
    2,
    18
  ],
  "outside": [
    6,
    6.07,
    6.5
  ],
  "liquid": [
    33,
    36.78,
    40
  ],
  "power": true,
  "mode": "H",
  "temp": 21,
  "fan": "A",
  "swingh": false,
  "swingv": false,
  "econo": false,
  "powerful": false,
  "comfort": false,
  "streamer": false,
  "sensor": false,
  "quiet": false
}

**Comment by revk** [comment]
The stats on that report are every minute (configurable I think) with min, average, and max. I think the manual explains that, but if I have missed it, I am happy to update it or take a pull request. There is an option to force it always to be an array and not be one value (when it has not changed), which again I think is in the manual.
